### PR TITLE
Seeder

### PR DESCRIPTION
**Pull request by davydhong**
seeded orientdb and postgres.
moving on with postgres based on latency measurements and usability.

next step: CRUD and k6 stress testing

### CODE REVIEW

File: server/api-v1/models/postgres.model.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const pool = require('../../../database/postgres/connection.postgres.js');
+const Promise = require('bluebird');
+
+const dbPool = function(queryString) {
```
Comment (Yontaekc) [inline server/api-v1/models/postgres.model.js:4]: like how u made function and reused

**Review by Yontaekc — APPROVED** [review]
Looks good to me davyduck!

File: dataSeed/oridentdb/album.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "config": { "haltOnError": false },
+  "source": { "file": { "path": "/Users/davidhong/Desktop/HackReactor/SDC/davy-album-player/dataGeneration/postgres/data/albumsAsync/data0.csv" } },
```
Comment (chrislarson) [inline dataSeed/oridentdb/album.json:3]: Hard-coded path

File: package.json [hunk 3]
```diff
@@ -12,15 +12,20 @@
     "mon-server": "nodemon server/index.js",
     "react-dev": "webpack -d --watch",
     "react-prod": "webpack --watch",
+    "gen-data": "node --max-old-space-size=4384 dataGeneration/postgres/dataGeneration.tables.js",
     "seed-data": "node seedDatabase.js",
-    "test": "jest"
+    "test": "jest",
+    "server-profile": "NODE_ENV=production node --prof server/index.js",
+    "curl-albums": "curl -X GET 'http://localhost:3001/api/v1/artists/100/albums/'",
```
Comment (chrislarson) [inline package.json:19]: Hard-coding localhost - might become problematic if you need to run this script on AWS

**Review by chrislarson — APPROVED** [review]
Nice work! Looks good to me 👍

File: dataSeed/oridentdb/album.json (hunk 2 above)
Comment (davydhong) [reply dataSeed/oridentdb/album.json:3]: this was for local testing of OrientDB.
should I leave it in or gitignore it?

**Review by Yontaekc — APPROVED** [review]

**Pull request merged**